### PR TITLE
CHI 2877 Implement Agent column in Teams view with labels

### PR DESCRIPTION
**Pull request by mythilytm**
## Description
This PR is for implementing the Agent column in the Teams View with sorting functionality by first name (previously by was sorted by activity) 
- `labels` property has been added as part of worker attributes in okta using the format of `stringarray`. They have been added to Annalise's and my profiles for testing. 
- It wasn't easy to accommodate an `object` with SAML in Okta, so each label is represented by a string with `:` as a delimiter. Each label string has information for abbreviation and full label text to shown with a tooltip. In output, there look like this: `"labels":"BL:Bilingual", "FR:French"`
Screenshot 2024-08-15 at 10 28 03 AM
- Refactored chip styling to accommodate all teams view chips (labels, active and disabled skills). For label, the 

### Checklist
- [x] Corresponding issue has been [opened](https://tech-matters.atlassian.net/browse/CHI-2877)
- [ ] New tests added
- [ ] Feature flags added
- [ ] Strings are localized
- [ ] Tested for chat contacts
- [ ] Tested for call contacts

### Other Related Issues

None

### Verification steps
Screenshot 2024-08-16 at 9 48 59 AM

### AFTER YOU MERGE

1. Cut a release tag using the Github workflow. Wait for it to complete and notify in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P

### CODE REVIEW

File: plugin-hrm-form/src/components/teamsView/SkillChip.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import React from 'react';
 import { Tooltip } from '@material-ui/core';
 
-import { SkillChipStyled, SmallCheckIcon, SmallDisabledIcon } from './styles';
+// import { SkillChipStyled, SmallCheckIcon, SmallDisabledIcon } from './styles';
```
Comment (stephenhand) [inline plugin-hrm-form/src/components/teamsView/SkillChip.tsx:20]: Is there a good reason to leave the commented out code here? We should probably either have a text comment explaining why the commented out code is here, or it should be removed

**Review by stephenhand — APPROVED** [review]
1 minor nit, otherwise LGTM

File: plugin-hrm-form/src/components/teamsView/SkillChip.tsx (hunk 1 above)
Comment (mythilytm) [reply plugin-hrm-form/src/components/teamsView/SkillChip.tsx:20]: It is not needed. I will delete it.

File: plugin-hrm-form/src/components/teamsView/AgentColumn.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+import React from 'react';
+import { WorkersDataTable, ColumnDefinition } from '@twilio/flex-ui';
+import Tooltip from '@material-ui/core/Tooltip';
+
+import { getAseloFeatureFlags } from '../../hrmConfig';
+import { AgentFullName, StyledChip } from './styles';
+
+export const setUpAgentColumn = () => {
+  if (!getAseloFeatureFlags().enable_teams_view_enhancements) return;
```
[the remark this answers is not in the document]
Comment (mythilytm) [reply plugin-hrm-form/src/components/teamsView/AgentColumn.tsx:27]: I will address this tomorrow during stand up: https://tech-matters.atlassian.net/browse/CHI-2905

**Pull request merged**